ormolu-live/cbits/init.c

```diff
@@ -0,0 +1,11 @@
+#include "Main_stub.h"
+#include <Rts.h>
+
+__attribute__((export_name("wizer.initialize"))) void __wizer_initialize(void) {
+  char *args[] = {"ormolu-live.wasm", "+RTS", "--nonmoving-gc"};
```

Comment:
Make sure to add a trailing `NULL`! The RTS expects the trailing `NULL` in the same way of libc main function, see https://en.cppreference.com/w/c/language/main_function

Reply:
Huh, thanks; I assumed it was not necessary as the length is passed in :facepalm: